scripts/CI_scripts/check_multitest.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+
+set -e
+
+projectPath=$(cd "$(dirname "${0}")" && pwd)
+
+function check_node_modules {
+   if [ ! -d "$projectPath/$1/scripts/node_modules" ];
+   then
+     cd "$projectPath/$1/scripts" && npm install
+     cp "$projectPath"/.env "$projectPath/$1/scripts"
+   fi
+}
+
+function check_multi_test {
+  cd "$projectPath/$1" && cargo update
+
+  if ! ERROR=$(cargo test);
```

Comment:
We need stderr as well

Reply:
We send only stderr to a slack channel.

Reply:
The cargo test always returns something for stdout, check $? instead. As @epanchee said, we need stderr, for e. g. using ERROR=$(cargo test 2>&1);